src/Input/Input.js

```diff
@@ -67,6 +67,12 @@ export const styleSheet = createStyleSheet('MuiInput', (theme) => {
         appearance: 'none',
       },
     },
+    multiLine: {
+      resize: 'none',
+      'line-height': 'inherit',
```

Comment:
So I tried this out. 

Here's what it looks like with inherit:

![screen shot 2017-04-09 at 2 47 20 pm](https://cloud.githubusercontent.com/assets/3895824/24840025/83b464de-1d33-11e7-95f7-1f4213be8d8e.png)

And here's what it looks like with 1: 

![screen shot 2017-04-09 at 2 47 31 pm](https://cloud.githubusercontent.com/assets/3895824/24840027/8aecdb1e-1d33-11e7-9859-66f4c0d76411.png)

Think it makes sense to stick with inherit for now